manager/integration/tests/common.py

```diff
@@ -12,6 +13,8 @@
 SIZE = str(16 * 1024 * 1024)
 VOLUME_NAME = "longhorn-testvol"
 DEV_PATH = "/dev/longhorn/"
+VOLUME_RWTEST_COUNT = 512
+VOLUME_RWTEST_START_POSITION = 0
```

Comment:
Make location randomly as well. It's optimal if we can check the whole disk but that may take too long for a 20G disk. So I think it's OK for us to compromise for now using random data/location.